dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,219 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
```

Comment:
we should be calling these openshift pipeline builds.  strictly speaking, jenkins is an implementation detail (though one that is a heavily leaked abstraction)